src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FeedbackAnalysisService, FeedbackDetail } from 'app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.service';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { AlertService } from 'app/core/util/alert.service';
+
+@Component({
+    selector: 'jhi-feedback-analysis',
+    templateUrl: './feedback-analysis.component.html',
+    standalone: true,
+    imports: [ArtemisSharedModule],
+    providers: [FeedbackAnalysisService],
+})
+export class FeedbackAnalysisComponent implements OnInit {
+    @Input() exerciseTitle: string;
+    @Input() exerciseId: number;
+    feedbackDetails: FeedbackDetail[] = [];
+
+    constructor(
+        private feedbackAnalysisService: FeedbackAnalysisService,
+        private alertService: AlertService,
+    ) {}
+
+    ngOnInit(): void {
+        this.loadFeedbackDetails(this.exerciseId);
+    }
+
+    async loadFeedbackDetails(exerciseId: number): Promise<void> {
```

Comment:
Follow-up: Does it make sense to have async/await and a Promise at the same time?